tests/EventBus/Middleware/SubscriberSpecificLoggingMiddlewareTest.phpt

```diff
@@ -84,7 +84,7 @@ class SubscriberSpecificLoggingMiddlewareTest extends DjTestCase
 
 		// expectations
 		$logger->shouldReceive('info')->once()->with('Event handling in subscriber started.', $expectedContext);
-		$logger->shouldReceive('error')->once()->with('Event handling in subscriber ended with error: some message', $expectedErrorContext);
+		$logger->shouldReceive('warning')->once()->with('Event handling in subscriber ended with error: some message', $expectedErrorContext);
```

Comment:
:thinking: I'm not sure about this "minor" change.
The original thought behind warning vs error was probably following: An exception thrown during command handling does not automatically mean an error - it may have been caused by some runtime problem, the exception is caught and handled by the caller. On the other hand a subscriber handling an event should "never" throw an exception, i.e. it's always an error.
Well, I guess you could let the exception from subscriber bubble all the way up and log it as an error on a different level, or let it kill the app.